### PR TITLE
[RW-12298] upgrade AoU docker images

### PR DESCRIPTION
**Pull request by yonghaoy**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/[ticket_number]

### What

-

### Why

-

### What to test

- 

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [ ] I validated this change 
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment

### CODE REVIEW

File: jenkins/gce-custom-images/prepare_gce_image.sh [hunk 1]
```diff
@@ -20,13 +20,13 @@ terra_jupyter_python="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-python:1.1.5"
...
 terra_jupyter_gatk="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:2.3.6"
-terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.8"
+terra_jupyter_aou="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.9"
 welder_server="us.gcr.io/broad-dsp-gcr-public/welder-server:8667bfe"
 openidc_proxy="broadinstitute/openidc-proxy:2.3.1_2"
 anvil_rstudio_bioconductor="us.gcr.io/broad-dsp-gcr-public/anvil-rstudio-bioconductor:3.18.0"
 
 # Note that this is the version used currently by AOU in production, the one above can be staged for testing
-terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.7"
+terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.8"
```
Comment (Qi77Qi) [inline jenkins/gce-custom-images/prepare_gce_image.sh]: which version we have in prod today? this needs to be what's in prod

Comment (yonghaoy) [reply jenkins/gce-custom-images/prepare_gce_image.sh]: good point. it is 2.2.7 now.

**Review by Qi77Qi — COMMENTED** [review]
seems dataproc was updated at some point https://github.com/DataBiosphere/leonardo/blob/develop/jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh#L24

**Review by Qi77Qi — APPROVED** [review]

**Comment by yonghaoy** [comment]
> seems dataproc was updated at some point https://github.com/DataBiosphere/leonardo/blob/develop/jenkins/dataproc-custom-images/prepare-custom-leonardo-jupyter-dataproc-image.sh#L24

The latest image on GCR is 2.2.9 https://console.cloud.google.com/artifacts/docker/broad-dsp-gcr-public/us/us.gcr.io/terra-jupyter-aou?project=broad-dsp-gcr-public
But the version should be 2.2.11 actually: https://github.com/DataBiosphere/terra-docker/pull/485/files
There might be some process is broken, not sure what happened

**Review by LizBaldo — APPROVED** [review]

**Comment by Qi77Qi** [comment]
```
java.lang.RuntimeException: google project could not be claimed for test, java.util.concurrent.TimeoutException: Future timed out after [5 minutes]
```

just triggered rerun of all tests

**Comment by yonghaoy** [comment]
> ```
> java.lang.RuntimeException: google project could not be claimed for test, java.util.concurrent.TimeoutException: Future timed out after [5 minutes]
> ```
> 
> just triggered rerun of all tests

` java.lang.RuntimeException: google project could not be claimed for test, java.util.concurrent.TimeoutException: Future timed out after [5 minutes]`
I don't think that is related

**Comment by yonghaoy** [comment]
> > ```
> > java.lang.RuntimeException: google project could not be claimed for test, java.util.concurrent.TimeoutException: Future timed out after [5 minutes]
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > just triggered rerun of all tests
> 
> ` java.lang.RuntimeException: google project could not be claimed for test, java.util.concurrent.TimeoutException: Future timed out after [5 minutes]` I don't think that is related

https://github.com/DataBiosphere/terra-resource-buffer/pull/294

**Comment by Qi77Qi** [comment]
I have a feeling we might run out of disk...but can verify Mon on a BEE

**Comment by yonghaoy** [comment]
> I have a feeling we might run out of disk...but can verify Mon on a BEE
Having issues with bee, forget about how to use it. 

For image size, seems not a big increase

- gce: 42GB to 44GB
- datapro 47 -> 47

**Comment by LizBaldo** [comment]
FYI @yonghaoy and @Qi77Qi I am taking a stab at updating the docker images again with a new bioconductor release and I will try to debug this while I am at it, I will keep you posted https://github.com/DataBiosphere/terra-docker/pull/488

**Comment by Qi77Qi** [comment]
thanks @LizBaldo !
haven't looked recently, but last time I looked, sth is preventing runtime from restarting

**Comment by LizBaldo** [comment]
Got it, the only thing that I remember doing back in March is fixing the AOU image this way. I do not think that would cause issues with restarting but I thought I would still share https://github.com/DataBiosphere/terra-docker/pull/476

**Comment by LizBaldo** [comment]
I will just point my BEE to this branch and see if I can create an AOU runtime and ssh into it

**Comment by LizBaldo** [comment]
We should probably follow up via slack, but here is what I found out on my BEE:

1- As Qi mentioned above, the issue is not when creating the runtime, but when resuming it
2- When sshing into the VM, I can confirm that the jupyter-server container is running, but the /home/jupyter directory is empty, and there are no jupyter server running, which is why the resume fails. I am not sure what could have caused this quite yet though.

I might need to decouple fixing this from releasing the new bioconductor and gatk images on Terra.
Screenshot 2024-05-16 at 4 10 18 PM
Screenshot 2024-05-16 at 4 10 31 PM
Screenshot 2024-05-16 at 4 10 45 PM
Screenshot 2024-05-16 at 4 11 25 PM

**Comment by LizBaldo** [comment]
@yonghaoy I think you can close this PR now, https://github.com/DataBiosphere/leonardo/pull/4566 will get you to the latest aou image available

**Comment by yonghaoy** [comment]
> @yonghaoy I think you can close this PR now, #4566 will get you to the latest aou image available

ha! I might just want to delete branch after closing..

**Pull request closed without merging**